site-src/blog/2022/first-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+description: >
+  We're pleased to announce the start of beta releases for the Gateway API
+  project with the release of version `v0.5.0`. This release includes `v1beta1`
+  versions of `GatewayClass`, `Gateway` and `HTTPRoute` as well as several other
+  significant improvements which this post will cover some of the highlights of.
+---
+
+# Introducing Gateway API v1beta1
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: June 22, 2022 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We're pleased to announce the start of beta releases for the Gateway API
+project with the release of version `v0.5.0`. This release includes `v1beta1`
+versions of `GatewayClass`, `Gateway` and `HTTPRoute` as well as several other
+significant improvements which this post will cover some of the highlights of.
+
+## What is Gateway API
+
+Gateway API is a collection of resources centered around `Gateway` resources
+(which represent the underlying network gateways / proxy servers) to enable
+robust Kubernetes service networking through expressive, extensible and
+role-oriented interfaces that are implemented by many vendors and have broad
+industry support.
+
+Originally conceived as a successor to the well known `Ingress` API, the
+benefits of Gateway API include (but are not limited to) explicit support for
+many commonly used networking protocols (e.g. `HTTP`, `GRPC`, `TCP`, `UDP`) as
+well as tightly integrated support for Transport Layer Security (TLS). The
+`Gateway` resource in particular enables implementations to manage the lifecycle
+of network gateways as a native Kubernetes API (whereas previously these might
+have been managed somewhere else, like helm).
+
+If you're an end-user interested in some of the benefits of Gateway API we
+invite you to jump in and find an implementation that suits you. At the time of
+this release there are over a dozen implementations for popular API gateways and
+service meshes and guides are available to start exploring quickly.
+
+### Getting Started
+
+The Gateway API project itself is a Kubernetes API like `Ingress` and you can
+think of the implementations of it as similar to the implementations for
+`IngressClass` or `StorageClass` in that Kubernetes itself has no default
+implementation of it (though your cluster provider might).
+
+Take a look at our [API concepts documentation][concepts] and check out some of
+our [Guides][guides] to start familiarizing yourself with the APIs and how they
```

Comment:
`guides` does not have a link definition anywhere.

Reply:
Thank you, fixed in 82421d13bd444b49980d66e8db97824026d5d053